test/api/services/freshtrakService.test.ts

```diff
@@ -34,6 +33,26 @@ describe('freshtrakService', () => {
       mockGetFreshTrakEvents.mockRestore();
     });
 
+    it("doesn't call the freshtrakAPIService when the site_id doesn't map to a FT agencyId", async () => {
+      const site_id = '-9999';
+      const zip = '43123';
+      const agencyId = 6
+
+      await getFTLocationData(site_id, zip);
+      // order of this test matters, must be before any successful tests
```

Comment:
The order won't matter if you put the setup functions inside `beforeEach` and `afterEach`. Then it will run before and after each `it` block.

Reply:
Thanks for the tip!